doc/source/parameters/dem/post-processing.rst

```diff
@@ -0,0 +1,53 @@
+Post-processing
+-------------------
+Lethe has built in post-processing capabilities. All post-processing results are written in a VTU file with the same name as the one chosen for the particles results plus the suffix ``-postprocess_data``. The post-processing subsection is according to the following example:
+
+.. code-block:: text
+
+ subsection post-processing
+  # Enable the use of calculate granular temperature and calculate particles' average velocity
+  set Lagrangian post-processing = false
+
+  # Enable writing grid VTU files
+  set write grid = true
+
+  # Enable calculate particles' average velocity
+  set calculate particles average velocity = false
+
+  # Enable calculate granular temperature
+  set calculate granular temperature = false
+
+  # Choose time step to start calculating particles average velocity and/or granular temperature
+  set initial step = 0
+
+  # Choose time step to stop calculating particles average velocity and/or granular temperature
+  set end step = 0
+
+  # Choose output frequency of particles average velocity and/or granular temperature calculation
+  set output frequency = 100000
+
+ end
+
+.. note::
+ The parameters default values are the ones presented above.
+
+* The ``Lagrangian post processing`` enables the use of ``calculate particles average velocity``, ``calculate granular temperature``, and ``write grid``.
+
+* The ``write grid`` parameter enables one to output the grid as VTU files. The output files can be used for post-processing. By default, it is ``true``. The files with the grid will have ``-grid`` in their names.
+
+* The ``calculate particles average velocity`` chooses whether the average velocity of the particles will be output or not. It generates a file with the average velocity of the particles.
```

Comment:
Can you explain a bit how the average velocity is calculated and how it is represented in the output? I don't understand how it works since it's exported on a grid.